src/navigationDrawer/navigationDrawer.component.tsx

```diff
@@ -178,6 +191,10 @@ class NavigationDrawer extends Component<CombinedNavigationProps> {
   }
 
   public render(): React.ReactElement {
+    const { plugins } = this.props;
+    const imgSrc = this.props.darkMode
+      ? plugins[0]?.logoDarkMode
+      : plugins[0]?.logoLightMode;
```

Comment:
The inclusion of question marks here prevented a bug from sometimes occurring which would make the page crash. This would be because the plugins were coming back as undefined, which could be related to #732. From multiple tests I've yet to see the logo fail to load when question marks are included and I don't see the harm in leaving them in.